.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -10,6 +10,10 @@ assignees: ""
 
 <!-- A clear and concise description of what the bug is. -->
 
+**Does this happen with only Scala Syntax and Metals enabled?**
```

Comment:
I would rephrase that to sth like: `Do you have other Scala-related extensions like dotty, lightbend sbt, etc. ?` I'm afraid that some users might have an impression that non-scala extensions may affect metals (sometimes they do like errors from java server, but it's a different case I think)

Reply:
But non-scala extension actually may also cause issues, there is a bunch of interactions that happened in the past and this might help out to rule them out.

Reply:
Do you expect that users will list all/some of their extensions when filling the form? I doubt it. On the contrary, it may be easier for them to list scala-related extensions. 

I'm afraid that such a general question won't have ay benefits and it's better to compile more narrowed one (like about scala-related extensions or java LSP) which will target 70% of cases than a very generic one which will require too much hassle from the users ("should I list all of them?", "I don't know if someone can interfere with Metals"). Don't you think?

>  there is a bunch of interactions that happened in the past

For example? I'm familiar with issues about java LSP interop, but I don't recall others.

Reply:
> If we don't disable all other extensions then we cannot be sure that there is an interaction.

It's true but in reality, it can be very impractical. 

Sorry for being very unconvinced about this change, but I'm really worried that it can be very misleading to normal users (not these few pro-users). 

Maybe we can refactor the issue template to be divided into a few parts like the one in metals is? Then we can make a section about other extensions non-mandatory or prefill it with some defaults or leave a suggestion through placeholders? What do you think?
It's not my intention to do nothing, but I want to do something which will have a minimal impact on creating an issue. Assuming that 60-70% of issues may come with java or scala related extensions it can be a good middle ground?